l/lit/lit_ubi_8.5.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+#----------------------------------------------------------------------------
+#
+# Package       : llvm-project
```

Comment:
this is misleading if we are not building the entire llvm-project tree, let's be specific